docs/en/getting_started.md

```diff
@@ -42,6 +43,7 @@ xrmocap
     └── body_models
         ├── gmm_08.pkl
         ├── smpl_mean_params.npz
+        ├── gmm_08.pkl (optional)
```

Comment:
There are two files with the same name in this layer. The file is not so big, I suggest that we remove the optional comment. Otherwise, one may find it is required when the whole estimation fails after a long matching.

Reply:
Optional comment has been removed.